app/assets/stylesheets/app/components/flip-grid.scss

```diff
@@ -120,7 +120,7 @@
     list-style: none;
     //
     &:hover {
-      background: $black;
+      background: rgba($black,.5);
```

Comment:
`.5` should be written with a leading zero as `0.5`<br>Commas in function arguments should be followed by a single space